pycontrails/core/met.py

```diff
@@ -707,7 +707,7 @@ def get(self, key: str, default_value: Any = None) -> Any:
     def __setitem__(
         self,
         key: Hashable | list[Hashable] | Mapping,
-        value: MetDataArray | xr.DataArray | np.ndarray | xr.Variable,
+        value: Any,
```

Comment:
Why the generalization of type here?

Reply:
For all instances of `value` except for `MetDataArray`, we simply call `xr.Dataset.__setitem__`. The signature there is `value: Any`, and I was noticing that we were being too strict here. For example, in the `ACCF` model, we had a line of code:

```
self.source[key] = (maCCFs.dim_order, arr.data)  # type: ignore
```

We don't allow tuple as a possible type of `value`, but `xr.Dataset.__setitem__` allows this.